ui/admin-portal/src/app/components/tasks/assign-tasks-candidate/assign-tasks-candidate.component.ts

```diff
@@ -99,30 +99,26 @@ export class AssignTasksCandidateComponent implements OnInit {
     if(task.name === 'duolingoTest') {
```

Comment:
Do we still need this code? If we have a specific button for assigning coupons.

Reply:
Yes, as of now assigning a task via the traditional method won't grant you a coupon. You will get the instructions that you have a coupon but there won't be any assigned you. This code triggers the assignment of coupon and task. 

If we only use the buttons to assign coupons I can hide the option to assign the task alone since that doesn't make much since and will cause confusion.